File: Client/ClientProject/Assets/Scripts/Data/CharacterData.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public enum CharacterClass
{
    Warrior = 0,
    Hunter = 1,
    Sorcere = 2,
}

public class CharacterData
{
    public int ID { get; set; }
    public CharacterClass Class { get; set; }

    /// <summary>
    /// 名字
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    /// 资源位置
    /// </summary>
    public string Resource { get; set; }
    /// <summary>
    /// 描述
    /// </summary>
    public string Description { get; set; }

    /// <summary>
    /// 基础书读
    /// </summary>
    public int BaseSpeed { get; set; }

    /// <summary>
    /// 基础生命
    /// </summary>
    public int BaseHp { get; set; }
    /// <summary>
    /// 基础魔法值
    /// </summary>
    public int BaseMp { get; set; }

    /// <summary>
    /// 基础力量
    /// </summary>
    public int BaseSTR { get; set; }
    /// <summary>
    /// 基础智力
    /// </summary>
    public int BaseINT { get; set; }

    /// <summary>
    /// 基础体力
    /// </summary>
    public int BaseBRA { get; set; }
    /// <summary>
    /// 基础敏捷
    /// </summary>
    public int BaseDEX { get; set; }
    /// <summary>
    /// 成长力量
    /// </summary>
    public float GrowthSTR { get; set; }
    /// <summary>
    /// 成长智力
    /// </summary>
    public float GrowthINT { get;set; }
    /// <summary>
    /// 成长体力
    /// </summary>
    public float GrowthBRA { get;set; }
    /// <summary>
    /// 成长敏捷
    /// </summary>
    public float GrowthDEX { get; set;}
    /// <summary>
    /// 基础暴击率
    /// </summary>
    public float BaseCrit { get; set; }
}

File: Server/Src/gameData/CharacterData.go
package gameData

type CharacterClass int

const (
	Class_Warrior CharacterClass = iota
	Class_Hunter
	Class_Sorcerer
)

type CharacterData struct {
	ID        int            // 表格ID
	Class     CharacterClass // 职业
	Name      string         // 名字
	BaseSpeed int            // 基础速度

	BaseHp  int // 基础生命
	BaseMp  int // 基础蓝量
	BaseSTR int // 基础力量
	BaseINT int // 基础智力
	BaseBRA int // 基础体力
	BaseDEX int // 基础敏捷

	GrowthSTR float32 // 成长力量
	GrowthINT float32 // 成长智力
	GrowthBRA float32 // 成长体力
	GrowthDEX float32 // 成长敏捷

	BaseCrit float32 // 基础暴击率
}
